src/helpers/metrics.ts

```diff
@@ -44,10 +44,12 @@ new client.Gauge({
 const totalMonthlyApiRequestsLimit = new client.Gauge({
   name: 'total_monthly_api_requests_limit',
   help: 'Monthly API requests limit per app',
-  labelNames: ['app']
+  labelNames: ['app', 'tier']
 });
-Object.entries(config.limits).forEach(([app, { monthly }]) => {
-  totalMonthlyApiRequestsLimit.set({ app }, monthly);
+Object.entries(config.limits).forEach(([app, tierLimit]) => {
+  Object.entries(tierLimit).forEach(([tier, limit]) => {
+    totalMonthlyApiRequestsLimit.set({ app, tier }, limit.monthly);
+  });
```

Comment:
@wa0x6e This should be fine right?

It will return these values:
```
total_monthly_api_requests_limit{app="snapshot-hub",tier="0"} 2000000
total_monthly_api_requests_limit{app="snapshot-hub",tier="1"} 5000000
total_monthly_api_requests_limit{app="score-api",tier="0"} 2000000
total_monthly_api_requests_limit{app="score-api",tier="1"} 5000000
```

Reply:
Yes